bulls_and_cows/lib/bulls_and_cows.ex

```diff
@@ -17,10 +15,33 @@ defmodule BullsAndCows do
             moves: [],
             win: false
 
+  @typedoc "Game state, should be passed between moves"
+  @type t :: %__MODULE__{
+          number_encrypted: BullsAndCows.encrypted_riddle(),
+          seed: integer(),
+          moves: list(BullsAndCows.move()),
+          win: boolean()
```

Comment:
Помогите читателю, ~~посадите дерево~~ покажите явно, что по этому ключу у вас булево значение.
```suggestion
          win?: boolean()
```